io/CMakeLists.txt

```diff
@@ -18,6 +18,16 @@ detray_add_library( detray_io io
 target_link_libraries( detray_io INTERFACE
    dfelibs::dfelibs nlohmann_json::nlohmann_json vecmem::core detray::core )
 
+# https://stackoverflow.com/questions/38823537/execute-git-describe-in-custom-target
+# Run git describe
+execute_process(COMMAND ${GIT_EXECUTABLE} describe --tags --always OUTPUT_VARIABLE GIT_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
This is only run on cmake configuration like this, right?

Reply:
@krasznaa might be able to answer

Reply:
For practical reasons it is usually a good idea to add the `--dirty` flag to `git describe`, because this will tell you if there were any uncommitted changes.

Reply:
```git describe``` does not print the correct tag anyway: v0.22.0-72-g8cdaae96 ...

Reply:
I adapted the vecmem code now